card/spec/lib/card/set/trait_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe Card::Set::Trait do
+  class Card
+    module Set
+      module Type
+        module Phrase
+          extend Card::Set
+          card_accessor :write, type: :pointer
+          card_accessor :read, type_id: Card::PointerID
```

Comment:
re Q2-4: It's only about testing `card_accessor`. I needed cards with codenames in the test db and we create`*write` with codename when we seed the test db. `read` was then the natural next candidate
re Q1: I can pass a symbol to type_id?